### PR TITLE
fix testing-framework

### PR DESCRIPTION
**Pull request by sallyom**
remove trace steps to isolate the path issue, then fix the path issue

### CODE REVIEW

**Comment by sallyom** [comment]
@Gregory-Pereira @cooktheryan @lmilbaum ptal, thanks!

File: .github/workflows/testing_framework.yaml [hunk 1]
```diff
@@ -53,50 +53,8 @@ jobs:
...
-          ./otelcol --config otel-collector-config.yaml &
-
-      - name: Start integration-tests trace
-        run: |
-          export WORKFLOW_NAME="testing_framework"
-          export STEP_NAME="integration-tests"
-          export TRACE_ACTION="start"
-          python ci/trace-steps.py
+      - name: Debug - List directory contents
+        run: ls -al ./recipes/natural_language_processing/chatbot
```
Comment (Gregory-Pereira) [inline .github/workflows/testing_framework.yaml]: Do we still need this?

**Review by Gregory-Pereira — APPROVED** [review]
/lgtm, just reccomend removing 1 debug step but either way it passes muster, thanks

**Review by lmilbaum — APPROVED** [review]
LGTM

File: .github/workflows/testing_framework.yaml (hunk 1 above)
Comment (sallyom) [reply .github/workflows/testing_framework.yaml]: we don't need it, i'll remove it

**Comment by sallyom** [comment]
@Gregory-Pereira I removed the debug statement and also re-commented the `on` section to run on PR

**Pull request merged**